common.mk

```diff
@@ -6,10 +6,11 @@ SHELL=/bin/bash
 #########################################################################################
 # variables to get all *.scala files
 #########################################################################################
-lookup_scala_srcs = $(shell find -L $(1)/ -name target -prune -o -iname "*.scala" -print 2> /dev/null)
+lookup_srcs = $(shell find -L $(1)/ -name target -prune -o -iname "*.$(2)" -print 2> /dev/null)
```

Comment:
Fancy :+1:

Reply:
Note: This just looks for `sv` sources. I assume we probably want to have it also be `v` sources.